lib/control/include/clingo/control/solver.hh

```diff
@@ -576,6 +576,15 @@ class SymbolTable {
 //! A unique pointer to a symbol table.
 using USymbolTable = std::unique_ptr<SymbolTable>;
 
+//! Event emitted by a solver after the program is grounded.
+// TODO: Simplify/Move on next output refactoring
+class Grounded : public Clasp::Event {
+  public:
+    explicit Grounded(Input::ProgramParamVec const &params)
+        : Event(this, subsystem_load, verbosity_quiet), params(params) {}
+    Input::ProgramParamVec const &params; // NOLINT
```

Comment:
I was consistently using pointers + maybe a getter to never store any refs. But maybe it is best to use a span here and just pass by value.

Reply:
Switched to std::span.